app/src/main/res/layout/activity_main.xml

```diff
@@ -56,6 +55,7 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:elevation="8dp"
+            android:visibility="gone"
```

Comment:
The dialog bottom sheet appeared above the bottom navigation, and it prevented you from changing tabs.